sidebarsDocsKits.js

```diff
@@ -407,6 +407,20 @@ const sidebars = {
                 },
             ],
         },
+        {
+            type: 'category',
+            label: 'Model Based Development and Data Processing (MDP) KIT',
+            link: {
+                type: 'generated-index',
+            },
+            collapsed: true,
+            items: [
+                {
+                    type: 'autogenerated',
+                    dirName: 'kits/Behaviour Twin MDP Kit',
```

Comment:
Can you please check that the preview of you kit will be shown under following pageRoute:

pageRoute: "/docs-kits/next/kits/...